README.adoc

```diff
@@ -78,25 +80,25 @@ All files marked "Copyright INRIA" in this distribution are copyright
 Informatique et en Automatique (INRIA) and distributed under the
 conditions stated in file LICENSE.
 
-INSTALLATION:
+== INSTALLATION
 
 See the file INSTALL for installation instructions on Unix, Linux and
 MacOS X machines.  For MS Windows, see README.win32.
 
-DOCUMENTATION:
+== DOCUMENTATION
 
 The OCaml manual is distributed in HTML, PDF, Postscript, DVI, and
 Emacs Info files.  It is available on the World Wide Web, at
 
-        http://caml.inria.fr/
+http://caml.inria.fr/
```

Comment:
Is a copy of the manual hosted on http://ocaml.org ? At least the link on the main page points to caml.inria.fr.